### PR TITLE
feat: created user models

### PR DESCRIPTION
**Pull request by ryanchen321**
#### **What does this PR do?**

Created the schema and exported the user models. Also created a new directory called models under src for future models. 

#### **How is this PR tested?**

Tested model by creating a test user based on the model. 

#### **Checklist**

-   [✔️] I have performed a self-review of my own code
-   [✔️] I have commented my code in hard-to-understand areas
-   [✔️] My changes generate no new warnings
-   [✔️] My merge commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary)

### CODE REVIEW

File: src/models/User.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { Schema, model, connect, Model } from 'mongoose';
```
Comment (sunny-zuo) [inline src/models/User.ts:1]:
```suggestion
import { Schema, model } from 'mongoose';
```

We don't use the `connect` or `Model` imports, so no point including them.

File: src/models/User.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+interface User {
+    name: string; 
+    email: string; 
+    password: string; 
+    registeredAt: Date; 
+    verified: Boolean;
```
Comment (sunny-zuo) [inline src/models/User.ts:8]:
```suggestion
    verified: boolean; 
```

For consistency, all primitive types should be lowercase. This is pretty annoying with mongoose schema expecting uppercase, but for everything non-schema related, we should use lowercase.

**Review by sunny-zuo — APPROVED** [review]
lgtm! I auto-formatted it with Prettier so that we can have consistent style :)

**Comment by ryanchen321** [comment]
Sounds good, thanks!

**Pull request merged**